ui/apps/dashboard/src/layout/sidebar.tsx

```diff
@@ -25,10 +30,15 @@ const Sidebar = () => {
       .map((m) => (m.handle as IRouteObjectHandle).sidebarKey);
   }, [matches]);
   return (
-    <div className={'w-full h-full overflow-y-auto'}>
+    <div
+      className={cn('w-full', 'h-full', 'overflow-y-auto', {
+        'collapsed-sidebar': collapsed,
+      })}
+      style={{ transition: 'width 0.3s' }}
```

Comment:
when render to the dom, the ant Menu component already has `transition: width 0.3s cubic-bezier(0.2, 0, 0, 1) 0s;` style
![image](https://github.com/user-attachments/assets/bdedde94-cf4a-4c45-94ba-9faf80ce3c85)